central/notifiers/microsoftsentinel/sentinel.go

```diff
@@ -170,13 +214,23 @@ func Validate(sentinel *storage.MicrosoftSentinel, validateSecret bool) error {
 		errorList.AddString("Log Ingestion Endpoint must be specified")
 	}
 
+	if sentinel.GetAuditLogDcrConfig().GetEnabled() {
+		if sentinel.GetAlertDcrConfig().GetDataCollectionRuleId() == "" {
+			errorList.AddString("Audit Logging Data Collection Rule Id must be specified")
+		}
+
+		if sentinel.GetAlertDcrConfig().GetStreamName() == "" {
+			errorList.AddString("Audit Logging Stream Name must be specified")
+		}
```

Comment:
Shouldn't these be `GetAuditLogDcrConfig()`?  Looks like copy/paste error.

Reply:
Yes! Good catch.